hack/build-images.sh

```diff
@@ -30,7 +30,7 @@ if [[ "${OS_RELEASE:-}" == "n" ]]; then
 
   echo "Building images from source ${OS_RELEASE_COMMIT}:"
 	echo
-  os::build::build_static_binaries "${OS_IMAGE_COMPILE_TARGETS[@]-}" "${OS_SCRATCH_IMAGE_COMPILE_TARGETS[@]-}"
+  OS_GOFLAGS="${OS_GOFLAGS:-} ${OS_IMAGE_COMPILE_GOFLAGS}" os::build::build_static_binaries "${OS_IMAGE_COMPILE_TARGETS[@]-}" "${OS_SCRATCH_IMAGE_COMPILE_TARGETS[@]-}"
```

Comment:
Ok, so just the one security question on tag / untag left for me.

On Fri, Jun 24, 2016 at 11:25 AM, Alexey Gladkov <notifications@github.com>
wrote:

> In hack/build-images.sh
> <https://github.com/openshift/origin/pull/8938#discussion_r68414976>:
>
> > @@ -30,7 +30,7 @@ if [[ "${OS_RELEASE:-}" == "n" ]]; then
> >
> >    echo "Building images from source ${OS_RELEASE_COMMIT}:"
> >  	echo
> > -  os::build::build_static_binaries "${OS_IMAGE_COMPILE_TARGETS[@]-}" "${OS_SCRATCH_IMAGE_COMPILE_TARGETS[@]-}"
> > +  OS_GOFLAGS="${OS_GOFLAGS:-} ${OS_IMAGE_COMPILE_GOFLAGS}" os::build::build_static_binaries "${OS_IMAGE_COMPILE_TARGETS[@]-}" "${OS_SCRATCH_IMAGE_COMPILE_TARGETS[@]-}"
>
> I know. There it was before my changes.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/8938/files/b065817711a716ae72b3590cc6aa139200b6b4ac..9317d4f3890a5a79628d1afcf3e2b77d5e7b33b8#r68414976>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p-beM7rIlFzM-uHZKsZ4KNMXGLEyks5qO_b6gaJpZM4IiLYZ>
> .
>